app/Http/Controllers/Order/InvoiceController.php

```diff
@@ -137,7 +137,14 @@ public function getInvoices(Request $request)
                         ->addColumn('user_id', function ($model) {
                             $first = $this->user->where('id', $model->user_id)->value('first_name');
                             $last = $this->user->where('id', $model->user_id)->value('last_name');
-                            $id = $this->user->where('id', $model->user_id)->first()->id;
+                            $id = $this->user->where('id', $model->user_id)->first();
+                            if ($id) {
+                                $id = $id->id;
+                            } else {
+                                $id = User::onlyTrashed()->find($model->user_id)->id;
+                                $first = User::onlyTrashed()->find($model->user_id)->first_name;
+                                $last = User::onlyTrashed()->find($model->user_id)->last_name;
```

Comment:
use negating condition for improving readability

Reply:
fixed